ketcher-autotests/tests/Macromolecule-editor/Macro-Micro-Switcher/macro-micro-switcher.spec.ts

```diff
@@ -0,0 +1,439 @@
+    */
+    await page.getByText('Hide').click();
+    await takePageScreenshot(page);
+    await page.getByText('Show Library').click();
+  });
+
+  test('Check that the Mol-structure opened from the file in Macro mode is visible on Micro mode', async ({
+    page,
+  }) => {
+    /* 
+    Test case: Macro-Micro-Switcher
+    Description: Mol-structure opened from the file in Macro mode is visible on Micro mode
+    */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await turnOnMicromoleculesEditor(page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Check that the Ket-structure opened from the file in Macro mode  is visible in Micro mode', async ({
+    page,
+  }) => {
+    /* 
+    Test case: Macro-Micro-Switcher
+    Description: Mol-structure opened from the file in Macro mode is visible on Micro mode when
+    */
+    await openFileAndAddToCanvas('KET/stereo-and-structure.ket', page);
+    await turnOnMicromoleculesEditor(page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Check that Zoom In/Zoom Out/ Reset Zoom Tools work (UI Buttons) after switching to Macro mode', async ({
+    page,
+  }) => {
+    /* 
+    Test case: Macro-Micro-Switcher
+    Description: Zoom In/Zoom Out/ Reset Zoom Tools work after switching to Macro mode
+    */
+    const numberOfPressZoomIn = 5;
+    const numberOfPressZoomOut = 8;
+    await openFileAndAddToCanvas(
+      'KET/three-monomers-connected-with-bonds.ket',
+      page,
+    );
+    await turnOnMicromoleculesEditor(page);
+    await turnOnMacromoleculesEditor(page);
+    for (let i = 0; i < numberOfPressZoomIn; i++) {
+      await waitForRender(page, async () => {
+        await page.getByTestId('zoom-in-button').click();
+      });
+    }
+
+    await takeEditorScreenshot(page);
+
+    for (let i = 0; i < numberOfPressZoomOut; i++) {
+      await waitForRender(page, async () => {
+        await page.getByTestId('zoom-out-button').click();
+      });
+    }
+    await takeEditorScreenshot(page);
+    await page.getByTestId('reset-zoom-button').click();
+    await takeEditorScreenshot(page);
+  });
+
+  test('Check that Zoom In/Zoom Out/ Reset Zoom Tools work (Mouse scroll) after switching to Macro mode', async ({
+    page,
+  }) => {
+    /* 
+    Test case: Macro-Micro-Switcher
+    Description: Zoom In/Zoom Out/ Reset Zoom Tools work after switching to Macro mode
+    */
+    const randomNegativeNumber = -80;
+    const randomPositiveNumber = 50;
+    const numberOfMouseWheelScroll = 5;
+    await openFileAndAddToCanvas(
+      'KET/three-monomers-connected-with-bonds.ket',
+      page,
+    );
+    await turnOnMicromoleculesEditor(page);
+    await turnOnMacromoleculesEditor(page);
+    await page.keyboard.down('Control');
+    for (let i = 0; i < numberOfMouseWheelScroll; i++) {
+      await page.mouse.wheel(0, randomNegativeNumber);
+    }
+    await page.keyboard.up('Control');
```

Comment:
Maybe you can create function with number parameter for:
`    await page.keyboard.down('Control');
    for (let i = 0; i < numberOfMouseWheelScroll; i++) {
      await page.mouse.wheel(0, randomNegativeNumber);
    }
    await page.keyboard.up('Control');`
and reuse it below?